packages/amplify-category-api/src/graphql-transformer/transform-graphql-schema-v2.ts

```diff
@@ -213,6 +218,19 @@ const buildAPIProject = async (
   return builtProject;
 };
 
+const getRDSLayerMapping = async (): Promise<RDSLayerMapping> => {
+  try {
+    const response = await fetch(LAYER_MAPPING_URL);
+    if (response.status === 200) {
+      const result = await response.json();
+      return result as RDSLayerMapping;
+    }
+  } catch (err) {
+    // Ignore the error and return default layer mapping
+  }
```

Comment:
Should we warn on non-200/failing fetch requests? So we at least have some indication in logs that the latest layer isn't being pulled.

Reply:
Added a warning message if CLI is unable to pull the latest layer config.